src/map/utils/charutils.cpp

```diff
@@ -2613,6 +2613,12 @@ namespace charutils
                 PChar->pushPacket(new CMessageBasicPacket(PChar, PChar, 0, 0, 0x200));
                 return;
             }
+
+            // Can't equip anything in sub slot if you're using H2H other than shields
```

Comment:
Eh? Comment confuses, because actual h2h can't use shield either. Only the no-skilled 1 hand punch can.

Reply:
when you equip the shield it unequips your h2h so you're bare-fisted with one punch

Reply:
the way it was worded is a lil wonky if that's what you meant to say, makes it sound like saying "if you are h2h, only a shield can be here". Maybe `// equipping a shield will remove h2h item, else sub slot must remain empty``

Reply:
updated cinnebt